packages/common/src/core/schemas/internal/ConfigurableEntity.ts

```diff
@@ -2,3 +2,8 @@
  * Generic interface for a Hub Entity
  */
 export type ConfigurableEntity = Record<string, any>;
+
+/**
+ * Generic interface for a layout card
```

Comment:
I think we could make these descriptions a bit more clear. It's honestly a bit difficult for me to even describe what they are, but I don't think this is necessarily accurate. Basically this is an interface defining what dynamic config options we can pass into the `getEditorConfig` function to help us build the `schema`/`uiSchema`.